app/controllers/api/v1/posts_controller.rb

```diff
@@ -39,6 +39,17 @@ def destroy
         end
       end
 
+      def like
+        like = @post.likes.find_or_initialize_by(user_id: current_user.id)
+        like.status = params[:status]
```

Comment:
It's counter intuitive that `status` param is related to `Like` model but not `Post`.